schema.sql

```diff
@@ -6,3 +6,6 @@ create table animals (
 	neutered boolean,
 	weight_kg decimal
 );
+
+alter table animals 
+	add species integer;
```

Comment:
- [ ] As stated in the project requirement you  should add a new column called species with **a string type** Just like the below  sample  ` ALTER TABLE animals ADD species varchar(255);`